documentation/articles/xtoolkit/remote.md

```diff
@@ -0,0 +1,125 @@
+# XToolkit Remote
+
+## Overview
+
+An opinionated HTTP library for Mobile Development. It provides a common way to make HTTP requests. It helps you to write more efficient code in mobile and desktop applications written in C#.
```

Comment:
Good question :)
It was borrowed from Fusillade library, currently maybe not so actual, because Fusillade not used by default).
Basically it "marketing" remark about features.